src/security-council-mgmt/SecurityCouncilUpgradeAction.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.16;
+
+import "./interfaces/IGnosisSafe.sol";
+import "./SecurityCouncilMgmtUtils.sol";
+
+/// @notice Action contract for updating security council members. Used by the security council management system.
+contract SecurityCouncilUpgradeAction {
+    address internal constant SENTINEL_OWNERS = address(0x1);
+
+    IGnosisSafe public immutable securityCouncil;
+
+    constructor(address _securityCouncil) {
+        securityCouncil = IGnosisSafe(_securityCouncil);
+    }
+
+    /// @notice updates members of security council multisig to match provided array
+    function updateMembers(address[] memory _updatedMembers) external {
+        // always preserve current threshold
+        uint256 threshold = securityCouncil.getThreshold();
+
+        address[] memory membersToAdd = new address[](6);
```

Comment:
Maybe better to use _updatedMembers.length / 2. Could also make this more general and make it _updateMembers.length?